pkg/manila/initcontainer.go

```diff
@@ -66,6 +68,7 @@ func InitContainer(init APIDetails) []corev1.Container {
 	envVars["DatabaseHost"] = env.SetValue(init.DatabaseHost)
 	envVars["DatabaseUser"] = env.SetValue(init.DatabaseUser)
 	envVars["DatabaseName"] = env.SetValue(init.DatabaseName)
+	envVars["Logging"] = env.SetValue(strconv.FormatBool(init.Logging))
```

Comment:
For now, until we remove the `init container` (that will be part of a dedicated patch), the easiest way to define different logging options between `manilaAPI` and `Scheduler` - `Share` is not a conditional in `manila.conf`.
`manila.conf` is inherited by the umbrella `Manila` CR, and it results the very same for all the underlying services.
Instead of making a temporary (complex) change in the Controller(s) logic, it would be a good idea handle this part in the init container with a simple way.
An additional boolean and the related logic in `init.sh` can be used to define which service we want to have the `logging.conf` config option.
By doing this we control where this change will land.